.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Deploy
```

Comment:
that could be changed - you build the project on github but it could be builded and deployed by netlify. All env varables and etc. could be defined there. Now we have a mix here - we deploy github-pages branch on netlify.

Reply:
You are probably right. it would be easier to keep it on Netlify but I really like this approach with the build step on Github Actions. Github Actions has a very good free tier plan (better than Netlify). In my opinion, keeping the entire building process here is a good lesson for creating CI/CD pipelines. We have the opportunity to check how the whole CI/CD process looks like (how many environments exist, etc.)

Reply:
aaa okey now i get it. It deploys gh-pages on netlify and triggered by Github actions